migration/sql-files/062-categories.sql

```diff
@@ -0,0 +1,23 @@
+CREATE TABLE categories (
+	created_at timestamp with time zone,
+	updated_at timestamp with time zone,
+	deleted_at timestamp with time zone,
+	id uuid primary key DEFAULT uuid_generate_v4() NOT NULL,
+	name text NOT NULL
+);
+
+CREATE INDEX categories_id_index ON categories (id);
+CREATE UNIQUE INDEX categories_name_idx ON categories (name) WHERE deleted_at IS NULL;
+
+CREATE TABLE workitemtype_categories (
```

Comment:
In order to be consistent, I'd say we have to rename this table to `work_item_type_categories`